include/Dialect/LWE/IR/LWEAttributes.td

```diff
@@ -63,12 +73,36 @@ def LWE_BitFieldEncoding
     Example:
 
     ```
-    #lwe_encoding = #lwe.polynomial_coefficient_encoding<cleartext_start=30, cleartext_bitwidth=3>
+    #lwe_encoding = #lwe.bit_field_encoding<cleartext_start=30, cleartext_bitwidth=3>
+    %lwe_ciphertext = arith.constant <[1,2,3,4]> : tensor<4xi32, #lwe_encoding>
+    ```
+  }];
+}
+
+def LWE_UnspecifiedBitFieldEncoding
```

Comment:
Why have a sep

Reply:
I assume the comment got caught off but was along the lines of "why not just make the cleartext_start optional/default to 31?" It seemed a little odd to check the presence of this value in determining if encoding params were set rather than separating the attributes, but having a c++ helper would help here.

Reply:
Sorry, just fat fingered enter :) Ignore this one.